src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -129,13 +152,64 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
   }, [state]);
 
   const onSubmit = useCallback(
-    async (vals: Partial<FormValues>) => {
-      if (!user) return;
+    async (
+      vals: Partial<FormValues>,
+      userStatuses: UserStatus[],
+      showUserStatuses: boolean
+    ) => {
+      if (!user || formError) return;
       try {
         // unfortunately the typing is off for react-hook-forms.
-        if (!!venueId)
-          await updateVenue({ ...(vals as VenueInput), id: venueId }, user);
-        else await createVenue(vals as VenueInput, user);
+        if (!!venueId) {
```

Comment:
```suggestion
        if (venueId) {
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/3d83fdf0a1c6006435e8500d063e921a7b822160